ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java

```diff
@@ -1492,6 +1492,42 @@ public void testRouterMetadataCacheDeleteCompositeBlob() throws Exception {
     }
   }
 
+  @Test
+  public void testRouterMetadataCacheUpdateTTLCompositeBlob() throws Exception {
+    try {
+      Properties properties = getNonBlockingRouterProperties("DC1");
+      properties.setProperty("router.blob.metadata.cache.enabled", Boolean.toString(true));
+      properties.setProperty("router.smallest.blob.for.metadata.cache", Long.toString(0));
+      setRouterWithMetadataCache(properties, new AmbryCacheMetrics(1,1,1,1));
```

Comment:
Sorry I was sick a couple days ago and didn't have enough time to review the unit test code, but I think the name of "AmbryCacheMetrics" is confusing, especially on "Metrics" part. 
We have a convention to use "Metric" name when it's actually the metrics, like histogram, timer, counter etc.. This object is not metrics, it's more of stats. I suggest we change the name to something like AmbryCacheStats? What do you think?

Reply:
I am fine with name but I'd prefer to do it in a separate follow up patch rather than this one. There would be many renames across many files.

Reply:
I mean I am fine with renaming, but in a follow up patch.